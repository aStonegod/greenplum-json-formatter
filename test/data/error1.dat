{ "id": 1, "t": "1abc", "f4": 1.4, "f8": 1.8 },
{ "idx": 2, "t": "2abc", "f4": 2.4, "f8": 2.8 },
{ "id": "3", "t": "3abc", "f4": 3.4, "f8": 3.8 },
{ "id": 4, "t": "4abc", "f4": 4.4, "f8": 4.8 },
{ "id": 5, "t": "5abc", "f4": "5.4", "f8": 5.8 },
{ "id": 6, "t": "6abc", "f4": 6.4, "f8": "6.8" },
{ "id": 7, "t": 1, "f4": 7.4, "f8": 7.8 }
