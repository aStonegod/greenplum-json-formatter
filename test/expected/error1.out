NOTICE:  Found 5 data formatting errors (5 or more input rows). Rejected related input data.
1|1abc|1.4|1.8
4|4abc|4.4|4.8
